client/packages_test.go

```diff
@@ -87,3 +89,45 @@ func (cs *clientSuite) TestClientPackages(c *check.C) {
 		},
 	})
 }
+
+const (
+	appID = "chatroom.ogra"
+)
+
+func (cs *clientSuite) TestClientPackageCallsEndpoint(c *check.C) {
+	_, _ = cs.cli.Package(appID)
```

Comment:
I wouldn't say there's a strong idiom about the particular suggestion I made, and it's okay if you want to keep as-is.

More generally, I would resist splitting tests too much just because you have more than one assertion in the same test, though. Having focused tests is good, but the number of assertions is not necessarily indicative of whether it's one aspect being tested or many.